lib/test/mdx_test.ml

```diff
@@ -385,3 +385,19 @@ module Predicate = struct
 
   let toploop = "toploop"
 end
+
+let run_exn_defaults =
```

Comment:
I think it would make more sense to keep this function scoped in the `dune-gen` command only, to keep the mdx_test api clean (otherwise we would need to document this function signature and precise that only the `dune-gen` command should call it), what do you think?

Reply:
I am not sure that this would be easily doable: we want this function to be part of the publicly installed `ocaml-mdx` lib so that is can be used by the independent executable whose code is generated by `dune-gen`.

Reply:
What I meant is that we could just write this function (in the verbatim string for generated code) in `dune-gen`, since this will only be used by dune-gen, and all the parameters used (packages, predicates) are exported as well as `run_exn` (so just call run_exn instead of run_exn_defaults, and add the missing args in the verbatim string).

Or we could also make most `run_exn` optional to have the same behavior as the `run_exn_defaults` you defined. If we do this that would be nice to document the default values for these args.

Reply:
You are right, I moved it to the generated code.